### PR TITLE
Overview card style tightening

### PR DESCRIPTION
**Pull request by thisisdano**
Addresses issues in https://github.com/18F/cg-dashboard/issues/846
**Paired with https://github.com/18F/cg-dashboard/pull/894 in `cg-dashboard`**

Works in org and space views as well.

- - -

screen shot 2016-12-30 at 9 48 45 am

### CODE REVIEW

File: src/css/components/all.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 @import 'billboard.scss';
 @import 'card.scss';
 @import 'confirm.scss';
+@import 'contents.scss';
```
Comment (thisisdano) [inline src/css/components/all.scss:7]: new contents styles apply to the overview home card and also to the overview panels in org and space views

File: src/css/components/card.scss [hunk 2]
```diff
@@ -25,3 +25,9 @@
 .card-items {
   padding: $grid-4 $grid-2;
 }
+
+.card-title-primary {
+  font-size: $sans-s8;
+  font-weight: 400;
+  line-height: $lh-single;
+}
```
Comment (thisisdano) [inline src/css/components/card.scss:33]: new style for primary card hed (the org name in the card)

File: src/css/components/contents.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+.contents-primary {
+  font-size: $sans-s7;
+  font-weight: 400;
+  line-height: $lh-single;
+  top: 2px;
+
+  .icon-bordered {
+    top: -2px;
+  }
+}
```
Comment (thisisdano) [inline src/css/components/contents.scss:14]: small `top` fixes to improve vertical alignment

File: src/css/components/contents.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+.contents-path {
+  color: $color-mediumgray;
+}
```
Comment (thisisdano) [inline src/css/components/contents.scss:30]: path is now gray

File: src/css/components/panel.scss [hunk 5]
```diff
@@ -155,8 +155,9 @@ h4 + .panel {
 .panel-row-boxed {
   background-color: $color-white;
   border: 1px solid $color-textblack;
-  margin-bottom: $panel-padding;
-  padding: $panel-padding;
+  border-radius: $rounded;
+  margin-bottom: $grid-2;
+  padding: 1.4rem $panel-padding;
```
Comment (thisisdano) [inline src/css/components/panel.scss:160]: fixed margins, border radius, and padding

File: src/css/components/panel.scss [hunk 6]
```diff
@@ -166,6 +167,10 @@ h4 + .panel {
     margin-bottom: 0;
     padding-bottom: $panel-padding;
   }
+
+  .panel-row {
+    padding-bottom: 0;
+  }
```
Comment (thisisdano) [inline src/css/components/panel.scss:173]: Corrects bottom padding in the overview card

**Review by thisisdano — COMMENTED** [review]
My comments on these changes

File: src/css/components/contents.scss [hunk 7]
```diff
@@ -0,0 +1,28 @@
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.contents-primary {
```
Comment (msecret) [inline src/css/components/contents.scss:4]: I'm not really sure what `contents` means in this context. What is this component trying to accomplish? What's it's purpose?

Comment (thisisdano) [reply src/css/components/contents.scss:4]: `contents` is the contents of an org or space. The contents styles are used in both the overview card (I should really rename this `home` to reduce confusion...) and the overview panel. I guess it isn't so much a `contents` component as much as common styles between two separate components.

I don't really care where the styles live, however. They don't need to be in this special file. Is there a place you'd prefer them?

Comment (thisisdano) [reply src/css/components/contents.scss:4]: I also don't care if it's named `contents` — any unique name is OK with me.

File: src/css/components/panel.scss (hunk 6 above)
Comment (line47) [reply src/css/components/panel.scss:173]: The padding on the bottom is causing the bottom border and icons to not have enough spacing. This impacting all `.panel-row-boxed .panel-row` 

screen shot 2016-12-30 at 1 57 39 pm

**Review by line47 — CHANGES_REQUESTED** [review]

**Comment by thisisdano** [comment]
@line47 ok, I'll get rid of it for now and figure this out later.

**Comment by thisisdano** [comment]
Actually, rewrote w/ `:last-child`.

screen shot 2016-12-30 at 11 08 53 am

**Comment by thisisdano** [comment]
ok, all fixed now. I think.
close enough, at least.

screen shot 2016-12-30 at 11 14 02 am

File: src/css/components/contents.scss (hunk 7 above)
Comment (msecret) [reply src/css/components/contents.scss:4]: I was trying to think what the best name will be

**Comment by msecret** [comment]
Just to be sure, make sure to check overview page, org overview, and space overview, as panel changes will usually effect those.

**Comment by thisisdano** [comment]
Looks good to me on those other pages

**Review by line47 — APPROVED** [review]

**Pull request merged**